### ISSUE TITLE
Clock problem

### ISSUE THREAD

**Issue by mboldin** [lichess-org/lichobile#936]
**I'm submitting a 
- bug report

# Bug report

**Current behavior:**

When playing with time control my time runs after I submit a move. This only seems to happen once my clock gets below 30 seconds.  It even runs despite a pre move or move ahead choice.  It seems pre moves are not recorded after I get down to about 30 seconds left.  The problem started about one week ago and happens on WiFi and data connections

**Expected behavior:**

Clock should not advance much if I have a valid pre move or a quick move 

**Steps to reproduce:**

Play 5-0.  Wait till clock turns red at 30 seconds.  Then any moves are not promply recorded 

**Environment information**
- phone hardware.   Pixel 3
- phone operating system version   Android version 9
- version of lichess mobile.  v6.1.0

**Comment by BradPenwarden** [comment]
I am also able to confirm this problem.

Phone Hardware: Moto G5
Operating system: Android 8.1.0 , Patch Oct1 2018
Lichess v7.1.0

**Comment by StuAngely** [comment]
https://lichess.org/rSUuZBglWlFm

Can also confirm this is happening. If you look at the timer at the end of this game I lose time during my opponents turn and his clock just stops going down.

**Comment by mboldin** [comment]
StuAngely-- your issue looks the same as mine. I lose time during what should be my opponents turn.  It only happens the last 30 seconds or so and once I get to 10 seconds I  never get a move recorded.

**Comment by mboldin** [comment]
The problem seems to have gone away for me. It was likely a server priblem. It was reported in the Lichess forums as happening for players on more than just the Android app.

**Comment by wsobasik** [comment]
That happend to me as well couple times per ~1k games.
I play 5 minutes games and here is the game: https://lichess.org/x7JF33G1

I moved a rock but my move didn't cause the change of the clock (to the opponents move) in effect I lost on time despite that did a move. 
After my clock run out I saw my opponents move straight away. So I assume he had to saw my move and he was able to replay to it but for me it was like the application was still waiting for me to move and was still counting my time. 
I don't think it relates to the device used but I can agree that it happens only when it is close to the end of the game. May be related to the games spectators(?)
